nemo/collections/asr/data/vocabs.py

```diff
@@ -17,10 +17,24 @@
 import string
 from typing import List
 
+import nltk
```

Comment:
Add this to our requirements

Reply:
nltk is g2p_en req, but I have added it also explicitly